tsqa/test_cases.py

```diff
@@ -46,7 +46,7 @@ class EnvironmentCase(unittest.TestCase):
     def run(self, result=None):
         unittest.TestCase.run(self, result)
         # we want to keep track of failures at a class level-- not instance level
-        self.__class__.__successful &= result.result.wasSuccessful()
+        self.__class__.__successful &= result.wasSuccessful()
```

Comment:
Hi @jacksontj , seems that it's `result.wasSuccessful()`
https://docs.python.org/3/library/unittest.html?highlight=unittest#unittest.TestResult.wasSuccessful

I manually add the following to test_example.py to run a single test:
```python
unittest = tsqa.utils.import_unittest()

if __name__ == '__main__':
  unittest.main()
```
And get exceptions as follows:
```
Traceback (most recent call last):
  File "test_example.py", line 239, in <module>
    unittest.main()
  File "/usr/lib/python2.6/site-packages/unittest2-0.8.0-py2.6.egg/unittest2/main.py", line 88, in __init__
    self.runTests()
  File "/usr/lib/python2.6/site-packages/unittest2-0.8.0-py2.6.egg/unittest2/main.py", line 237, in runTests
    self.result = testRunner.run(self.test)
  File "/usr/lib/python2.6/site-packages/unittest2-0.8.0-py2.6.egg/unittest2/runner.py", line 162, in run
    test(result)
  File "/usr/lib/python2.6/site-packages/unittest2-0.8.0-py2.6.egg/unittest2/suite.py", line 87, in __call__
    return self.run(*args, **kwds)
  File "/usr/lib/python2.6/site-packages/unittest2-0.8.0-py2.6.egg/unittest2/suite.py", line 126, in run
    test(result)
  File "/usr/lib/python2.6/site-packages/unittest2-0.8.0-py2.6.egg/unittest2/suite.py", line 87, in __call__
    return self.run(*args, **kwds)
  File "/usr/lib/python2.6/site-packages/unittest2-0.8.0-py2.6.egg/unittest2/suite.py", line 126, in run
    test(result)
  File "/usr/lib/python2.6/site-packages/unittest2-0.8.0-py2.6.egg/unittest2/case.py", line 649, in __call__
    return self.run(*args, **kwds)
  File "build/bdist.linux-x86_64/egg/tsqa/test_cases.py", line 52, in run
AttributeError: 'TextTestResult' object has no attribute 'result'
```